src/main/java/ab/common/core/proxy/CommonProxy.java

```diff
@@ -10,6 +10,7 @@
 import ab.common.lib.register.FlowerRegister;
 import ab.common.lib.register.ItemListAB;
 import ab.common.minetweaker.MineTweakerConfig;
+import ab.nei.*;
```

Comment:
wildcard import bad, please configure properly your IDE to avoid them.

Reply:
The internal wildcard imports are used because I use all the contents of that file / package